frontend/src/component/menu/Header/OldHeader.tsx

```diff
@@ -0,0 +1,333 @@
+import { useState, type VFC } from 'react';
+import useMediaQuery from '@mui/material/useMediaQuery';
+import { useTheme } from '@mui/material/styles';
+import { Link } from 'react-router-dom';
+import {
+    AppBar,
+    IconButton,
+    Tooltip,
+    styled,
+    type Theme,
+    Box,
+} from '@mui/material';
+import MenuIcon from '@mui/icons-material/Menu';
+import SettingsIcon from '@mui/icons-material/Settings';
+import UserProfile from 'component/user/UserProfile';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import MenuBookIcon from '@mui/icons-material/MenuBook';
+import { ReactComponent as UnleashLogo } from 'assets/img/logoDarkWithText.svg';
+import { ReactComponent as UnleashLogoWhite } from 'assets/img/logoWithWhiteText.svg';
+import { ReactComponent as CelebatoryUnleashLogo } from 'assets/img/unleashHoliday.svg';
+import { ReactComponent as CelebatoryUnleashLogoWhite } from 'assets/img/unleashHolidayDark.svg';
+
+import { DrawerMenu } from './DrawerMenu/DrawerMenu';
+import useUiConfig from 'hooks/api/getters/useUiConfig/useUiConfig';
+import { flexRow, focusable } from 'themes/themeStyles';
+import { NavigationMenu } from './NavigationMenu/NavigationMenu';
+import { getRoutes, getCondensedRoutes } from 'component/menu/routes';
+import DarkModeOutlined from '@mui/icons-material/DarkModeOutlined';
+import KeyboardArrowDown from '@mui/icons-material/KeyboardArrowDown';
+import LightModeOutlined from '@mui/icons-material/LightModeOutlined';
+import { filterByConfig, mapRouteLink } from 'component/common/util';
+import { useId } from 'hooks/useId';
+import { ThemeMode } from 'component/common/ThemeMode/ThemeMode';
+import { useThemeMode } from 'hooks/useThemeMode';
+import { Notifications } from 'component/common/Notifications/Notifications';
+import { useAdminRoutes } from 'component/admin/useAdminRoutes';
+import InviteLinkButton from './InviteLink/InviteLinkButton/InviteLinkButton';
+import { useUiFlag } from 'hooks/useUiFlag';
+import { Badge } from '../../common/Badge/Badge';
+
+const HeaderComponent = styled(AppBar)(({ theme }) => ({
```

Comment:
this code will be gone so it's better to give it a name prefixed with Old. I prefer to keep the history for the new component called Header.